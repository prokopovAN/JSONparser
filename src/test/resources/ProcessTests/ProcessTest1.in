key1.sub1 = 3
key1.sub2 = 1
key2 = 2
key2.sub3 = 4