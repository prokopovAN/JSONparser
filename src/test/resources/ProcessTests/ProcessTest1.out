{
  "key1": {
    "sub1": {
      "value": 3
    },
    "sub2": {
      "value": 1
    }
  },
  "key2": {
    "sub3": {
      "value": 4
    },
    "value": 2
  }
}